MCU Applications/Simple ATM Machine Project/Application/ATM ECU/AtmECU.h

```diff
@@ -0,0 +1,156 @@
+/*****************************************************************************
+* Module: Atm Module
+* File Name: AtmECU.c
+* Description: Header file for Atm Module
+* Author: Mohamed Magdy
+* Date: 28-July-2021
+******************************************************************************/
+
+
+#ifndef CARDAPP_H_
+#define CARDAPP_H_
+
+/*- INCLUDES -----------------------------------------------*/
+#include "../MCAL/Dio Driver/Dio.h"
+#include "../MCAL/Spi Driver/Spi.h"
+#include "../MCAL/Uart Driver/Uart.h"
+#include "../ECUAL/Lcd Driver/Lcd.h"
+#include "../ECUAL/Keypad Driver/Keypad.h"
+#include "../ECUAL/Lm35 Driver/Lm35.h"
+#include "../ECUAL/Motor Driver/Motor.h"
+#include "../ECUAL/Eeprom Driver/EEPROM_24.h"
+#include "../MCAL/Interrupt Driver/ISR.h"
+/*- CONSTANTS ----------------------------------------------*/
+#define CardRequestSignal_Pin		PORTC_PIN2_ID
+
+#define atmMode_size					6
```

Comment:
it's better to put macros definition between brackets, for example #define atmMode_size	(6).

Reply:
yes, you are definitely right, it's safer to do so. Thank you for your notice.